### DISCUSSION TITLE
Setting a date format

### DISCUSSION THREAD

**Discussion by lucasmalara** [Droid-ify/client#347] [Ideas]
Hello,
I'd appreciate if you add an option to change a date format that is displayed for versions of application.
For instance, if you set an **English** language, your only option is **mm/dd/yyyy** which can be confusing.
Having possibility to set a date format to **dd/mm/yyyy** or **yyyy/mm/dd** regardless of application language would be a cool thing.

**Comment by SebiderSushi** [comment]
As a workaround, android uses a `yyyy-mm-dd` date with the `en_CA` Locale [English (Canada)].
